chart/testdata/frobnitz/templates/wordpress.jinja.schema

```diff
@@ -0,0 +1,69 @@
+info:
+  title: Wordpress
+  description: |
+    Defines a Wordpress website by defining four replicated services: an NFS service, an nginx service, a wordpress-php service, and a MySQL service.
+
+    The nginx service and the Wordpress-php service both use NFS to share files.
+
+properties:
+  project:
+    type: string
+    default: dm-k8s-testing
+    description: Project location to load the images from.
+  nfs-service:
+    type: object
+    properties:
+      ip:
+        type: string
+        default: 10.0.253.247
```

Comment:
We have a wordpress example [here](https://github.com/kubernetes/deployment-manager/tree/master/examples/wordpress). Can we reuse it instead of building a new one?